apps/client/src/components/criminals/criminals.tsx

```diff
@@ -3,15 +3,14 @@ import { CriminalsCard, Category, Divider } from '../';
 import { Tooltip } from 'react-tooltip';
 import { GiPirateFlag, GiPirateGrave } from 'react-icons/gi';
 import { IconContext } from 'react-icons';
-import {
-  Criminal,
-  useGetAllRegularCriminalsQuery,
-  useGetAllPersonaNonGratasQuery,
-} from '../../store';
+import { Criminal, useGetAllCriminalsQuery } from '../../store';
 
 export const Criminals: React.FC = () => {
-  const { data: CriminalsData } = useGetAllRegularCriminalsQuery();
-  const { data: PersonaNonGratasData } = useGetAllPersonaNonGratasQuery();
+  const { data: CriminalsData } = useGetAllCriminalsQuery();
+  const regularCriminals =
```

Comment:
I believe this data doesn't stay constant because Criminals can be invalidated. (when adding a criminal for example)
`useEffect` might be more appropriate here